cargo-pgrx/src/command/init.rs

```diff
@@ -298,7 +369,13 @@ fn configure_postgres(pg_config: &PgConfig, pgdir: &PathBuf, init: &Init) -> eyr
     for var in PROCESS_ENV_DENYLIST {
         command.env_remove(var);
     }
-
+    // Work around the fact that pg16 requires icu to be installed and available
+    // via pkg-config, but doesn't add it to the pkg-config path. (Does a decent
+    // number of checks to try to avoid causing trouble when things were
+    // actually fine)
+    if cfg!(target_os = "macos") && pg_config.major_version().unwrap_or(0) >= 16 {
```

Comment:
can we make this `== 16`?  Just thinking that assuming what 17 will do might hide a problem from future us.  I'd probably rather 17 fail too when we first add support for it.

Reply:
Done.